reddit_liveupdate/hooks.py

```diff
@@ -0,0 +1,35 @@
+from pylons import c
```

Comment:
nit: "hooks.py" just sounds like it'd become a dumping ground.  would it be possible to name this module something related to the specific feature? (there's already other hooks in this plugin too that live in their own modules)